src/program.js

```diff
@@ -1,45 +1,54 @@
 /* @flow */
+import path from 'path';
+import {readFileSync} from 'fs';
 import yargs from 'yargs';
 
+import * as defaultCommands from './cmd';
 import {WebExtError} from './errors';
 
 
 /*
- * Pseudo-class for all global program options.
- *
- * This is used to validate the definition of command handlers.
- * Each class instance variable is a camel case expanded program
- * option. Each option is defined in program.yargs.option(...).
- */
-export class ProgramOptions {
```

Comment:
I removed this because it wasn't working (too many errors I could not resolve). I filed https://github.com/mozilla/web-ext/issues/106 to look for a solution.